backend/src/cron/tasks.ts

```diff
@@ -120,12 +120,18 @@ export const tokensSupplyCheck: RegularCheckFn = {
   ),
 };
 
+export const gasUsedHistoryCheck: RegularCheckFn = {
+  description: "gasUsedHistoryCheck",
+  fn: publishOnChange(
+    "gasUsedHistory",
+    queryGasUsedAggregatedByDate,
+    config.intervals.checkAggregatedStats
+  ),
+};
+
```

Comment:
Before this we have a `retriable` method wrapper for such queries because they could fail for some reason. How this case will be handled now?